server/server.js

```diff
@@ -136,6 +137,10 @@ server.setup = async (config) => {
     issuer: 'Yeep',
   });
 
+  const mail = new MailService({
```

Comment:
I believe this can be rewritten in simpler format.

```javascript
const mail = new MailService(config.mail);
```

Reply:
I just do this to copy these by value in case any mutation happens. I know it doesn't deep copy but i am currently doing this by muscle memory only